examples/nextjs-spa/src/pages/registration.tsx

```diff
@@ -65,23 +71,14 @@ const Registration: NextPageWithLayout = () => {
 
     // If ?flow=.. was in the URL, we fetch it
     if (flowId) {
-      getRegistrationFlow(String(flowId || "")).catch(
-        (error: AxiosError) =>
-          error.response?.status === 410 ?? createRegistrationFlow(returnTo),
-        // if the flow is expired, we create a new one
-      )
+      getFlow(String(flowId || "")).catch(() => createFlow(returnTo))
       return
     }
 
     // Otherwise we initialize it
-    createRegistrationFlow(returnTo)
-  }, [
-    createRegistrationFlow,
-    getRegistrationFlow,
-    router.isReady,
-    flowId,
-    returnTo,
-  ])
+    createFlow(returnTo)
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I'm not sure if we should disable linter rules like this.
Since the functions are memoized through the `useCallback` function, it should be alright to add them to this dependency list.